src/server/components/lists/controllers/index.ts

```diff
@@ -92,11 +121,40 @@ export function listsGetController(req: Request, res: Response): void {
   }
 
   const { serviceType } = params;
+  let { languagesProvided, servicesProvided } = params;
 
   let questionsSequence: QuestionName[];
   let partialPageTitle: string = "";
+  let partialPageHintText: string = "";
   let partialToRender: string = "";
   let error: boolean | QuestionError = false;
+  let partialData: QuestionDataSet[] | QuestionData[];
+  let languagesRows, languageNamesProvided, serviceNamesProvided;
+  let backUrl: string = "";
+
+  if (languagesProvided) {
+    const cleanedLanguagesProvided = cleanLanguagesProvided(languagesProvided as string);
+    languagesProvided = cleanedLanguagesProvided;
+    params.languagesProvided = cleanedLanguagesProvided ?? undefined;
```

Comment:
This line to me looks a bit odd. If `cleanedLanguagesProvided` is `null` or `undefined`, then make `params.languagesProvided` `undefined`.

What would be the disadvantage of doing this?

```suggestion
    params.languagesProvided = cleanedLanguagesProvided;
```

Reply:
fair enough, no benefit here.  I've changed this per suggestion.

Reply:
I don't think there is a need for the intermediary cleanedLanguagesProvided? 
```
  //not let - do not destructure
  const languagesProvided = cleanLanguagesProvided(params.languagesProvided)
```